packages/reports/src/Containers/statement.jsx

```diff
@@ -52,31 +52,21 @@ const DetailsComponent = ({ message = '', action_type = '' }) => {
     );
 };
 
-const statementws_href = urlFor('user/statementws', { legacy: true });
-
 const getRowAction = row_obj => {
     let action;
     if (row_obj.id && ['buy', 'sell'].includes(row_obj.action_type)) {
+        const unsupported_contract_type = extractInfoFromShortcode(row_obj.shortcode).category.toUpperCase();
         action =
-            getSupportedContracts()[extractInfoFromShortcode(row_obj.shortcode).category.toUpperCase()] &&
+            getSupportedContracts()[unsupported_contract_type] &&
```

Comment:
This is not needed and why is this called `unsupported`? shouldn't be the `supported` contracts?

Reply:
Maybe it'll be better to rename it to `contract_type` as we check if this contract is supported or not in lines 60-63?